Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-09-12
Initial Package Version: 1.60
Origin: Rawhide
Description: Fixes IPX Netstat

diff -Naur net-tools-1.60.orig/lib/ipx_gr.c net-tools-1.60/lib/ipx_gr.c
--- net-tools-1.60.orig/lib/ipx_gr.c	2000-10-28 10:59:42.000000000 +0000
+++ net-tools-1.60/lib/ipx_gr.c	2003-09-12 22:41:54.000000000 +0000
@@ -64,7 +64,7 @@
 	    continue;
 
 	/* Fetch and resolve the Destination */
-	(void) ap->input(5, net, &sa);
+	(void) ap->input(1, net, &sa);
 	strcpy(net, ap->sprint(&sa, numeric));
 
 	/* Fetch and resolve the Router Net */
diff -Naur net-tools-1.60.orig/netstat.c net-tools-1.60/netstat.c
--- net-tools-1.60.orig/netstat.c	2001-04-15 14:41:17.000000000 +0000
+++ net-tools-1.60/netstat.c	2003-09-12 22:41:55.000000000 +0000
@@ -1412,13 +1412,13 @@
 	}
 
 	/* Fetch and resolve the Source */
-	(void) ap->input(4, sad, &sa);
+	(void) ap->input(0, sad, &sa);
 	safe_strncpy(buf, ap->sprint(&sa, flag_not), sizeof(buf));
 	snprintf(sad, sizeof(sad), "%s:%04X", buf, sport);
 
 	if (!nc) {
 	    /* Fetch and resolve the Destination */
-	    (void) ap->input(4, dad, &sa);
+	    (void) ap->input(0, dad, &sa);
 	    safe_strncpy(buf, ap->sprint(&sa, flag_not), sizeof(buf));
 	    snprintf(dad, sizeof(dad), "%s:%04X", buf, dport);
 	} else
diff -Naur net-tools-1.60.orig/version.h net-tools-1.60/version.h
--- net-tools-1.60.orig/version.h	1970-01-01 00:00:00.000000000 +0000
+++ net-tools-1.60/version.h	2003-09-12 22:41:55.000000000 +0000
@@ -0,0 +1 @@
+#define RELEASE "net-tools 1.60"
